modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1265,8 +1265,7 @@ primary cluster’s IPs, separator by `,' |`nan`
 |System.CrossRegionReplication.PrimaryKafkaPort |Kafka mirrormaker
 primary cluster’s KafkaPort |`30002`
 
-|System.CrossRegionReplication.TopicPrefix |The prefix of GPE/GUI/GSQL
-Kafka Topic, by default is empty. |`nan`
+|System.CrossRegionReplication.TopicPrefix |The prefix used for replicated internal topics in the DR cluster. It determines whether the original or replicated topics (with the "primary." prefix) are used. |`nan`
```

Comment:
@anjaliIthapeTG Is "primary." the prefix for the original topics?
It's a little confusing, because the clause "(with the "primary." prefix)" is located after "replicated topics" instead of after "original [topics]".